app/_hub/kong-inc/openid-connect/index.md

```diff
@@ -14,7 +14,7 @@ description: |
   - Signed [JWT][jwt] access tokens ([JWS][jws])
   - Opaque access tokens
   - Refresh tokens
-  - Authorization code  
```

Comment:
@lena-larionova There were various changes like this where space was trimmed; please let me know if you would like me to isolate in a separate commit or revert these entirely.